README.md

```diff
@@ -66,6 +66,7 @@ encoding: UTF-8
 display-name: CodecCsv
 validate-length: true
 publish-header: false
+th2.csv.override_message_type: "message_type_prop_name"
```

Comment:
The property - is a property in message's metadata. It is not a part of the configuration.
It should work the following way:
1. we receive a raw message
2. we create a parsed message for csv content
3. if the raw message contains property `th2.csv.override_message_type` we should take its value and use as a message type for that message
4. if the raw message does not contain this property the default (the one that is currently used) message type should be used